src/app/custom-navigation-mode/custom-navigation-mode-factory.provider.ts

```diff
@@ -0,0 +1,13 @@
+import { BaseNavigationModeFactory, WizardComponent, NavigationMode } from 'angular-archwizard';
+import { CustomNavigationMode } from './custom-navigation-mode';
+
+export class CustomNavigationModeFactory extends BaseNavigationModeFactory {

```

Comment:
Maybe add some comments to this class too. I know that it should be clear why this class is required when applying your custom navigation mode, but I think it is better to be a bit excessive with the comments than not accidentally missing something.

Reply:
Makes sense. It looks particularly strange because the `CustomNavigationMode` class is documented, but `CustomNavigationModeFactory` isn't.

Reply:
Comments added in 212d1fd.